search/README.md

```diff
@@ -7,6 +7,11 @@
 
 Amundsen Search service serves a Restful API and is responsible for searching metadata. The service leverages [Elasticsearch](https://www.elastic.co/products/elasticsearch "Elasticsearch") for most of it's search capabilites.
 
+By default, it creates in total 3 indexes:
```

Comment:
I assume you mean `It could support three types of search index` ? I don't think it is default though as not every company indexed all three types of entities. WDYT?

Reply:
@feng-tao not really. I've been looking for this information (exactly default indexes) over the docs for some time but I couldn't find it. So I asked within slack chat: https://amundsenworkspace.slack.com/archives/CGFBVT23V/p1623066324225500
@NiravLangaliya and Mariusz Górski provided this information